script/dev

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This is a script for running Compose inside a Docker container. It's handy for
+# development.
+#
+# $ ln -s `pwd`/script/dev /usr/local/bin/docker-compose
+# $ cd /a/compose/project
+# $ docker-compose up
+# 
+
+set -e
+DIR="$(dirname "$(readlink "$0")")"/..
+docker build -t docker-compose $DIR
```

Comment:
So I guess the tradeoff here (vs using an official image that gets pulled on the first run) is that it's probably a bit faster the first run, but a bit slower on subsequent runs.  Even with the cache, it's still going to take a second or two to run each time, right?

That seems ok for a first pass, but we should really consider moving to just pulling an image from the docker registry.